mux_httpserver_test.go

```diff
@@ -28,7 +28,7 @@ func TestSchemeMatchers(t *testing.T) {
 		if resp.StatusCode != 200 {
 			t.Fatalf("expected a status code of 200, got %v", resp.StatusCode)
 		}
-		body, err := ioutil.ReadAll(resp.Body)
+		body, err := io.ReadAll(resp.Body)
```

Comment:
`ioutil` -> `io` is a valid change but let's not do it here. We can have another issue+PR for the same. Documentation PRs often don't touch the code parts.

Reply:
FYI #17 is created for the same

Reply:
done